reloj-checador/app/assets/stylesheets/checkins.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Checkins controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
Is this really needed?

Reply:
Deleted,indeed it is no needed; and otherwise the file is created again.